tests/integration/test_agent.py

```diff
@@ -15,16 +15,11 @@
 )
 from openhands.events.observation.browse import BrowserOutputObservation
 from openhands.events.observation.delegate import AgentDelegateObservation
-from openhands.runtime import get_runtime_cls
-
-TEST_RUNTIME = os.getenv('TEST_RUNTIME')
```

Comment:
ditto - i think we shouldn't hard code it?

Reply:
Resolved